firmware/cmake/generate_cubemx.mk

```diff
@@ -1,13 +1,14 @@
+space := $(subst ,, )
+
 ifeq ($(OS),Windows_NT)
 # Convert windows backslash to regular slash
 	CUBEMX_PATH := $(subst \,/,$(shell where STM32CubeMX))
+	CUBEMX_JAVA := $(dir $(subst $(space),\$(space),$(CUBEMX_PATH)))jre/bin/java
 else # Linux / MacOS
```

Comment:
Linux is installed in the same path as windows, see https://github.com/macformula/racecar/pull/196

Your `if` should check for MacOS and have windows/linux be the else

Reply:
Note: `$(OS)` on MacOS returns an empty string so need to check with `$(shell uname)`.